frontend/packages/dev-console/src/components/EmptyState.tsx

```diff
@@ -31,12 +31,15 @@ const Item: React.FC<ItemProps> = ({
       // eslint-disable-next-line react-hooks/rules-of-hooks
       .map((descriptor) => useAccessReview({ namespace, ...descriptor }))
       .every((x) => x);
+
+  const resolvedUrl = url ? url.replace(':namespace', namespace) : undefined;
```

Comment:
Good idea.
URL by definition of the type is always defined.
What about using a stricter regex replacement such as this:
```suggestion
  const resolvedUrl = url.replace(/:namespace\b/g, namespace);
```

Currently we do not show this page within the `all namespaces` selection. So we are ok to ensure that there is always a valid namespace option for replacement.